<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildForm</class>
 <widget class="QWidget" name="BuildForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BuildForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <item row="24" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Table of contents</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLineEdit" name="username_lineEdit"/>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QLineEdit" name="date_lineEdit"/>
   </item>
   <item row="23" column="2">
    <widget class="QCheckBox" name="table_of_contents_checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Server credentials</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dokuwiki server</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Margin</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="pagepath_listWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pagepath_browse_button">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dokuwiki password</string>
     </property>
    </widget>
   </item>
   <item row="26" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QLineEdit" name="margin_lineEdit">
     <property name="text">
      <string>1in</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="project_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>PDF document settings</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="QLineEdit" name="title_lineEdit"/>
   </item>
   <item row="21" column="2">
    <widget class="QLineEdit" name="languages_lineEdit"/>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Keywords</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Languages</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="notebook_folder_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="notebook_folder_browse_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="20" column="2">
    <widget class="QLineEdit" name="keywords_lineEdit"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Dokuwiki username</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="server_lineEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Notebook settings</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QLineEdit" name="author_lineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Notebook folder</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Pagepaths</string>
     </property>
    </widget>
   </item>
   <item row="28" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="generate_pdf_Button">
       <property name="text">
        <string>Generate pdf document</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upload_to_server_Button">
       <property name="text">
        <string>upload to server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_from_server_pushButton">
       <property name="text">
        <string>delete from server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="27" column="0">
    <widget class="QPushButton" name="generate_Button">
     <property name="text">
      <string>Save configuration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>notebook_folder_browse_button</tabstop>
  <tabstop>pagepath_browse_button</tabstop>
  <tabstop>server_lineEdit</tabstop>
  <tabstop>username_lineEdit</tabstop>
  <tabstop>password_lineEdit</tabstop>
  <tabstop>title_lineEdit</tabstop>
  <tabstop>author_lineEdit</tabstop>
  <tabstop>date_lineEdit</tabstop>
  <tabstop>keywords_lineEdit</tabstop>
  <tabstop>languages_lineEdit</tabstop>
  <tabstop>margin_lineEdit</tabstop>
  <tabstop>table_of_contents_checkBox</tabstop>
  <tabstop>generate_pdf_Button</tabstop>
  <tabstop>upload_to_server_Button</tabstop>
  <tabstop>notebook_folder_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>select_notebook_path()</slot>
 </slots>
</ui>
